applications/nrf_desktop/src/hw_interface/Kconfig.selector

```diff
@@ -7,6 +7,7 @@
 menu "Selector configuration"
 
 config DESKTOP_SELECTOR_HW_ENABLE
+	depends on CAF
```

Comment:
why?

Reply:
Because `selector_hw.c` includes files that are part of CAF. I didn't want to split those for now as this is just a porting PR.